Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -1,43 +1,65 @@
 import demistomock as demisto
 from CommonServerPython import *
 import time
-import requests
 from requests import Response
 
+import urllib3
+from urllib3 import exceptions as urllib3_exceptions
+
 
 class AuthError(Exception):
     pass
 
 
+def str_arg(args, name):
+    return str(args.get(name, ""))
+
+
+def int_arg(args, name):
+    i = args.get(name, None)
+    if i is not None:
+        return int(i)
+    return None
+
+
+def bool_arg(args, name):
+    return argToBoolean(args.get(name, False))
+
+
+def list_arg(args, name):
+    return argToList(args.get(name, []))
+
```

Comment:
These functions are using those you mentioned internally, this is just for readability. By the way I noted that the demisto openapi codegen tool we were formerly using was only converting args that were not numbers, i.e was using str(), argToBoolean and argToList but not arg_to_number (even if the argument type in the openapi was "integer"). I'm not sure if it's intentional but I will add use of arg_to_number just in case.